### PR TITLE
Import keys using secp256k1 private keys.

### PR DESCRIPTION
**Pull request by Daniel-VDM**
Given a secp256k1 private key, like the one generated from the following command:

```
openssl ecparam -genkey -name secp256k1 -text -noout -outform DER | xxd -p -c 1000 | sed 's/41534e31204f49443a20736563703235366b310a30740201010420/PrivKey: /' | sed 's/a00706052b8104000aa144034200/\'$'\nPubKey: /'
```

You can import the key with an optional name and passphrase. For example:
```
./hmy keys import-private-key b8798ca0a56ce16517ea37c6b1229cbb67cf0e022c423b044fe8f537830d8be5 foo-bar-baz --passphrase
```

### CODE REVIEW

File: cmd/subcommands/keys.go [hunk 1]
```diff
@@ -91,10 +91,30 @@ func keysSub() []*cobra.Command {
...
+	cmdImportSK := &cobra.Command{
+		Use:   "import-private-key <secp256k1_PRIVATE_KEY> [ACCOUNT_NAME]",
+		Short: "Import an existing keystore key (only accept secp256k1 private keys)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			passphrase := c.DefaultPassphrase
+			if userProvidesPassphrase {
+				passphrase = doubleTakePhrase()
+			}
+			userName := ""
+			if len(args) == 2 {
```
Comment (fxfactorial) [inline cmd/subcommands/keys.go]: Looking into letting `cobra` handle some of this logic for you/enforce, see 		`Args:  cobra.ExactArgs(1),` in `cmd/subcommands/balance.go:46`

File: pkg/account/import.go [hunk 2]
```diff
@@ -13,17 +15,32 @@ import (
...
+		name = generateName() + "-imported"
 	}
-	keyJSON, readError := ioutil.ReadFile(keypath)
-	if readError != nil {
-		return "", readError
+	privateKeyBytes, err := hex.DecodeString(privateKey)
+	if err != nil {
+		return "", err
+	}
+	if len(privateKeyBytes) != 32 {
```
Comment (fxfactorial) [inline pkg/account/import.go]: magic number 32 should be const, put in `pkg/common/values.go`

File: pkg/account/import.go [hunk 3]
```diff
@@ -13,17 +15,32 @@ import (
...
+		return "", ErrBadKeyLength
 	}
+
+	// btcec.PrivKeyFromBytes does not return an error
```
Comment (fxfactorial) [inline pkg/account/import.go]: Comment should explain why it does not return error

File: test.sh [hunk 4]
```diff
@@ -4,32 +4,41 @@ source ../harmony/scripts/setup_bls_build_flags.sh
 
 set -eiu
```
Comment (fxfactorial) [inline test.sh]: remove this file's changes from the PR

File: cmd/subcommands/keys.go (hunk 1 above)
Comment (Daniel-VDM) [reply cmd/subcommands/keys.go]: added arg range check

File: pkg/account/import.go (hunk 2 above)
Comment (Daniel-VDM) [reply pkg/account/import.go]: done

File: pkg/account/import.go (hunk 3 above)
Comment (Daniel-VDM) [reply pkg/account/import.go]: clarified comment

File: test.sh (hunk 4 above)
Comment (Daniel-VDM) [reply test.sh]: done

**Pull request merged**